### DISCUSSION TITLE
Out of stock SF6W

### DISCUSSION THREAD

**Discussion by AtMostafa** [Karpova-Lab/pyControl-D-Series-Breakout#5] [Q&A] [answered]
The [recommended microcontroller](https://store.micropython.org/product/PYBD-SF6-W4F2) has been out of stock for a while and I cannot find another supplier.
Do we know if the similar [SF2W](https://store.micropython.org/product/PYBD-SF2-W4F2) which is still in stock would work with the series-d PyControl?

If anyone knows of a supplier of SF6Ws, I would be grateful as well.

**Comment by alustig3** [comment, marked as answer]
Any of the Pyboard D-series boards (SF2W, SF3W or SF6W) will work with this breakout and pyControl.

**Reply by AtMostafa** [reply]
That's what I thought. Thanks, Andy!
